### PR TITLE
Headless chrome creation script and readme file

### PR DESCRIPTION
**Pull request by harold-wang**
*Issue #, if available:*

*Description of changes:*
We added script and readme file to generate the headless chrome for puppeteer.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by zhongnansu** [comment]
any idea why the github action fails?If a re-run will fix the issue, could you add a step retry? @joshuali925 @harold-wang

**Comment by joshuali925** [comment]
> any idea why the github action fails?If a re-run will fix the issue, could you add a step retry? @joshuali925 @harold-wang

because #228 is not merged, it will update chromium download url

File: kibana-reports/rendering-engine/headless-chrome/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## Commands to create headless_chrome
+Run below command to create headless_shell for each platform
+​
+headless-chrome.sh chrome-version-SHA (arch_name (arm64))
+. Mac x64: ./build_headless_chrome.sh <chrome-version-SHA>
+. Linux x64: ./build_headless_chrome.sh <chrome-version-SHA>
+. Linux arm64: ./build_headless_chrome.sh <chrome-version-SHA> arm64
+​
+# How to call in Command line:
+    >> please use what you used for testing this << replace me >>
```
Comment (joshuali925) [inline kibana-reports/rendering-engine/headless-chrome/README.md]: what's the purpose of these two lines?

File: kibana-reports/rendering-engine/headless-chrome/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    swiftshader
+      |-libEGL.so
+      |-libEGL.so.TOC
+      |-libGLESv2.so
+      |-libGLESv2.so.TOC
+# Additional libaries:
+  libnss3-dev
+  fonts-liberation
+  libfontconfig
+  (installation command example : sudo apt update && apt install libnss3-dev fonts-liberation libfontconfig)
```
Comment (joshuali925) [inline kibana-reports/rendering-engine/headless-chrome/README.md]: These dependencies are for Ubuntu, for CentOS I had to install a lot more
```
sudo yum install -y libnss3.so xorg-x11-fonts-100dpi xorg-x11-fonts-75dpi xorg-x11-utils xorg-x11-fonts-cyrillic xorg-x11-fonts-Type1 xorg-x11-fonts-misc fontconfig freetype
```
I listed them in this PR #222, could you update it here?

File: kibana-reports/rendering-engine/headless-chrome/README.md (hunk 1 above)
Comment (anirudha) [reply kibana-reports/rendering-engine/headless-chrome/README.md]: this was a review comment/ @harold-wang can you pls update here. thanks

Comment (harold-wang) [reply kibana-reports/rendering-engine/headless-chrome/README.md]: > what's the purpose of these two lines?
Josh, these are command used to verify chrome in command line. 
MAC/Linux PNG : ./headless_shell --headless --disable-gpu --screenshot=test.png https://www.google.com
MAC/Linux PDF: ./headless_shell --headless --disable-gpu --print-to-pdf=test.png https://www.google.com

Comment (anirudha) [reply kibana-reports/rendering-engine/headless-chrome/README.md]: thanks, saw the commits

**Review by harold-wang — COMMENTED** [review]
Add test command to verify chrome in linux:
MAC/Linux PNG : ./headless_shell --headless --disable-gpu --screenshot=test.png https://www.google.com
MAC/Linux PDF: ./headless_shell --headless --disable-gpu --print-to-pdf=test.png https://www.google.com

**Review by anirudha — APPROVED** [review]
thanks

**Review by joshuali925 — APPROVED** [review]
thanks

**Comment by zhongnansu** [comment]
> Add test command to verify chrome in linux:
> MAC/Linux PNG : ./headless_shell --headless --disable-gpu --screenshot=test.png https://www.google.com
> MAC/Linux PDF: ./headless_shell --headless --disable-gpu --print-to-pdf=test.png https://www.google.com

Np:  can we use the domain we owned such as opendistro website, instead of google.com

**Comment by harold-wang** [comment]
> > Add test command to verify chrome in linux:
> > MAC/Linux PNG : ./headless_shell --headless --disable-gpu --screenshot=test.png https://www.google.com
> > MAC/Linux PDF: ./headless_shell --headless --disable-gpu --print-to-pdf=test.png https://www.google.com
> 
> Np: can we use the domain we owned such as opendistro website, instead of google.com

image

**Review by zhongnansu — APPROVED** [review]

**Pull request merged**